### PR TITLE
Optimize next compiler for local development and make next js script production ready 

### PR DESCRIPTION
**Pull request by rahmatagungj**
## **Problem Description**
As a developer i need to increase productivity and development speed, also if looking as a user i need to load script more faster. 

### **How ​​this problem is currently solved**
- Creating custom defered next js script
- Add more configuration for local development

### **Why is this important to solve?**
- Make developer more than happier
- User can access bellshade web faster

## **Technical Aspects That Need Attention**
- Some configuration need to adjust with our timeline

## **QA**

N/A

## **Other Decisions**

N/A

### CODE REVIEW

**Review by reacto11mecha — CHANGES_REQUESTED** [review]

File: pages/_document.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            property="og:description"
+            content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia"
+          />
+          <meta
+            property="og:image"
+            content="https://cdn.statically.io/img/media.discordapp.net/w=800/attachments/891222600422936616/892682527720996884/unknown.png"
+          />
+          <meta property="og:image:width" content="800" />
+          <meta property="og:image:height" content="856" />
+          <meta name="twitter:card" content="summary_large_image" />
```
[the remark this answers is not in the document]
Comment (reacto11mecha) [reply pages/_document.js:57]: Oiya kelupaan, untuk yang preconnect gapapa, dipakein aja

[the remark this answers is not in the document]
Comment (rahmatagungj) [reply pages/_document.js]: refactored @reacto11mecha with branch fix/identity-refactor

**Review by reacto11mecha — APPROVED** [review]
Sip LGTM :+1:  Terimakasih atas kontribusinya

**Review by hasheemi — APPROVED** [review]

**Review by nomadkode — APPROVED** [review]

**Review by dhafitf — APPROVED** [review]

**Review by dipras — APPROVED** [review]
LGTM
Meluncur Deploy :rocket:

**Pull request merged**